mlir/include/mlir/Dialect/ArmSME/IR/ArmSMEOps.td

```diff
@@ -76,6 +76,7 @@ def TileSliceLayout : I32EnumAttr<"TileSliceLayout", "Layout of a tile slice", [
 def ArmSME_TileSliceLayoutAttr : EnumAttr<ArmSME_Dialect, TileSliceLayout,
                                           "layout"> {
   let assemblyFormat = "`<` $value `>`";
+  let defaultValue = "TileSliceLayout::Horizontal";
```

Comment:
Ah, this means that:

```
DefaultValuedAttr<ArmSME_TileSliceLayoutAttr,
                      "::mlir::arm_sme::TileSliceLayout::Horizontal">:$layout
```

can be replaced with:

```
    ArmSME_TileSliceLayoutAttr:$layout
```
?

Reply:
Yup, that's done in this patch, see below